Containerfile

```diff
@@ -1,19 +1,14 @@
 # SPDX-License-Identifier: Apache-2.0
-
-FROM quay.io/projectquay/golang:1.22 AS makefile
-
-RUN dnf install -qy diffutils && dnf clean all
+ARG TARGETOS
+ARG TARGETARCH
+ARG TARGETPLATFORM
+FROM golang:1.22 AS builder
```

Comment:
"--platform=$TARGETPLATFORM " was added to a FROM line below. Is that needed here too?

Reply:
Yes, I'll add it there as well